tests/test_validators.py

```diff
@@ -78,6 +81,38 @@ def test_repr(self):
         ) == repr(v)
 
 
+class TestMatchesRe(object):
+    """
+    Tests for `matches_re`.
+    """
+
+    def test_in_all(self):
+        """
+        Verify that this validator is in ``__all__``.
+        """
+        assert matches_re.__name__ in validator_module.__all__
+
+    def test(self):
```

Comment:
Please add a [test docstring](https://jml.io/pages/test-docstrings.html) and it'd also make sense to break the test up a little bit.

Reply:
broke it into two pieces with more semantic names and docstrings

Reply:
I really like the guidelines in that link you provided -- especially dropping the do-nothing words like "check this" or "test that"

along those lines though, the word "Tests" and "Verify" each occur 9x in docstrings in test_validators.py

should I follow the pattern, or not add any more instances of "tests" or "verify" in docstrings?